### PR TITLE
server can acquire webmks ticket

### PR DESCRIPTION
**Pull request by timogoebel**
This patch allows to acquire a webmks ticket for a server. This can be used with https://www.vmware.com/support/developer/html-console/.

### CODE REVIEW

File: lib/fog/vsphere/models/compute/tickets.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+              new(
+                service.vm_acquire_ticket(
+                  opts.merge(
+                    'instance_uuid' => server.instance_uuid
+                  )
+                )
+              )
+            else
+            raise 'server is required to create a ticket'
+          end
```
Comment (plribeiro3000) [inline lib/fog/vsphere/models/compute/tickets.rb]: Maybe you should just raise an exception if is not a `Fog::Compute::Vsphere::Server` to be more explicit?

**Comment by plribeiro3000** [comment]
Other then a code design comment LGTM.

**Comment by timogoebel** [comment]
@plribeiro3000: Thanks for the review. Good point. I followed your suggestion.

**Comment by plribeiro3000** [comment]
👍

**Comment by plribeiro3000** [comment]
Not a problem @jrgarcia . =)

**Comment by timogoebel** [comment]
Thanks @jrgarcia and @plribeiro3000.

@jrgarcia: If you don't mind and have some time, could you release a new version of fog-vsphere?

Sorry for hijacking this threat, but it's worth a shot: We're currently wondering if we can use the "[VMware HTML5 Console SDK](https://www.vmware.com/support/developer/html-console/)" inside of Foreman. Foreman is licensed unter GNU GPL v3. But we're not sure about the Console SDK. Do you happen to know anything about that or have anyone you can ask?

**Comment by jrgarcia** [comment]
@timogoebel I'll release a new version when I get a chance today. I have no idea, but I am on the SDK team at VMware, so i'm sure I can find someone with details. I'll send you an email if I find anything.

**Comment by jrgarcia** [comment]
Sorry I didn't get to it yesterday! [1.12.0](https://rubygems.org/gems/fog-vsphere/versions/1.12.0) has your change in it though.

**Pull request merged**